src/components/DatahubRequired/DatahubRequired.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { InjectAppServices } from '../../services/pure-di';
+import { injectIntl } from 'react-intl';
+
+// TODO: fix content and styles
+
+/**
+ * @param { Object } props
+ * @param { import('../../services/pure-di').AppServices } props.dependencies
+ */
+function DatahubRequired({ dependencies: { appConfiguration } }) {
+  return (
+    <section className="container-reports">
+      <h3>Datahub Reports</h3>
+      <p>
+        This feature is not active, please{' '}
```

Comment:
maybe this put in language messages file

Reply:
Yes, but I still do not know what to do with it. @mfantini did not answer me.

Reply:
Fixed!